lib/src/screens/initial_setup/widgets/initial_setup_welcome.dart

```diff
@@ -99,75 +97,81 @@ SizedBox initialSetupWelcome(BuildContext context) {
 /// This function constructs a `Row` widget designed to display a single piece
 /// of information in a profile UI. It is primarily used for laying out text-based
 /// information such as names, titles, or other key details in the profile section.
+///
+/// // comment out the following function as it is not used in the current version
+// of the app, anushka might need to use to in the future so keeping it here.
 
-Row buildInfoRow(String profName) {
-  return Row(
-    mainAxisAlignment: MainAxisAlignment.spaceBetween,
-    children: <Widget>[
-      Text(
-        profName,
-        style: TextStyle(
-          color: Colors.grey[800],
-          letterSpacing: 2.0,
-          fontSize: 17.0,
-          fontWeight: FontWeight.bold,
-          fontFamily: 'Poppins',
-        ),
-      ),
-    ],
-  );
-}
+// Row buildInfoRow(String profName) {
+//   return Row(
+//     mainAxisAlignment: MainAxisAlignment.spaceBetween,
+//     children: <Widget>[
+//       Text(
+//         profName,
+//         style: TextStyle(
+//           color: Colors.grey[800],
+//           letterSpacing: 2.0,
+//           fontSize: 17.0,
+//           fontWeight: FontWeight.bold,
+//           fontFamily: 'Poppins',
+//         ),
+//       ),
+//     ],
+//   );
+// }
 
 /// Builds a row widget displaying a label and its corresponding value.
 ///
 /// This function creates a [Column] widget containing a [Row] with two text elements:
 /// one for the label and the other for the profile name. It's used to display
 /// information in a key-value pair format, where `labelName` is the key and
 /// `profName` is the value.
+///
+/// // comment out the following function as it is not used in the current version
```

Comment:
How does this affect the doc string? It may look somewhat odd in the generated docs?